src/app/types/Table.ts

```diff
@@ -40,8 +41,22 @@ export interface TableFilterDef {
   multiple?: boolean;
   exhaustive?: boolean;
   cleared?: boolean;
+  singleDatePicker?: boolean;
```

Comment:
This is specific date/time properties, they should be in this type I think but in a sub-type like DateRangeTableFilterDef.
What is weird is that we didn't need it for the Date filter why now?

Reply:
the date range provide more properties to control the component that we can use to customize. ( it's not mandatory)